app/assets/src/components/views/Landing.jsx

```diff
@@ -56,6 +56,9 @@ class Landing extends React.Component {
         }
       })
       .then(() => {
+        // Although this event does not follow current naming conventions for
+        // frontend events, we keep it for continuity. See
```

Comment:
I just realized this event probably doesn't work anymore if we're not loading Segment on the landing page.. it could be moved to a backend event in HomeController#sign_up

Reply:
good catch. I confirmed none of those events have been logged lately. I will move it to the backend.